packages/x-date-pickers/vitest.config.jsdom.mts

```diff
@@ -0,0 +1,33 @@
+import { mergeConfig } from 'vitest/config';
+// eslint-disable-next-line import/no-relative-packages
+import filterReplacePlugin from '../../test/vite-plugin-filter-replace.mts';
+import sharedConfig from '../../vitest.shared.mts';
+import { getTestName } from '../../scripts/getTestName.mts';
+
+export const filterReplace = filterReplacePlugin(
+  [
+    {
+      filter: /\/AdapterDateFnsV2\/.*/,
+      replace: {
+        from: /from 'date-fns/g,
+        to: "from 'date-fns-v2",
```

Comment:
This is super nice and clean! 💙 💯 
Maybe we can already remove the `babel-plugin-replace-imports@1.0.2.patch` and the plugin itself together with the `babel` config? 🤔